docs/build-on-polkadot/xcm/asset-transfer-api/reference.md

```diff
@@ -0,0 +1,301 @@
+
+### Methods
+
+#### createTransferTransaction
+
+This function allows you to create XCM transactions to transfer assets or native tokens between different chains.
+
+It takes several parameters to specify the transfer details, including the destination chain, recipient address, assets to be transferred, and their amounts, as well as an optional parameter for further customization. It can infer what kind of transaction is necessary given the inputs. When sending cross-chain transfers, the API performs extensive validation to ensure the inputs are valid and the assets either exist or don't.
+
+!!! note
+	The `createTransferTransaction` function is designed to be a utility that simplifies the creation of the transaction. It does not sign or submit the created transaction on the blockchain. It simply generates the transaction in the requested format (e.g., payload, call, or submittable). After obtaining the transaction from the `createTransferTransaction` function, you will need to handle the signing and submission process separately.
+
+```ts
+public async createTransferTransaction<T extends Format>(
+  	destChainId: string,
+  	destAddr: string,
+  	assetIds: string[],
+  	amounts: string[],
+  	opts: TransferArgsOpts<T> = {},
+): Promise<TxResult<T>>
+```
+
+??? interface "Request parameters"
+
+	`destChainId` ++"string"++ <span class="required" markdown>++"required"++</span>
+	
+	ID of the destination chain ('0' for Relay chain, other values for parachains)
+
+	---
+
+	`destAddr` ++"string"++ <span class="required" markdown>++"required"++</span>
+
+	Address of the recipient account on the destination chain
+
+	---
+
+	`assetIds` ++"string[]"++ <span class="required" markdown>++"required"++</span>
+
+	Array of asset IDs to be transferred
+      
+    If a token or asset ID is provided as input, the API will resolve to using the `tokens` pallet. When no asset is passed in, the API will default to using the `balances` pallet.
+
+	---
+
+	`amounts` ++"string[]"++ <span class="required" markdown>++"required"++</span>
+
+	Array of amounts corresponding to each asset in `assetIds`
+
+	---
+
+	`opts` ++"TransferArgsOpts<T>"++
+
+    --8<-- 'code/build-on-polkadot/xcm/asset-transfer-api/reference/transfer-arg-opts.md'
+
+??? interface "Response parameters"
+
+    ++"Promise<TxResult<T>"++
+
+    --8<-- 'code/build-on-polkadot/xcm/asset-transfer-api/reference/tx-result.md'
+
+??? interface "Example"
+
+	***Request***
+
+	```ts
+	--8<-- 'code/build-on-polkadot/xcm/asset-transfer-api/reference/ctt-example-request.ts'
+	```
+
+	***Response***
+
+	--8<-- 'code/build-on-polkadot/xcm/asset-transfer-api/reference/ctt-example-response.md'
+
+#### claimAssets
+
+Creates a local XCM transaction to retrieve trapped assets. This function can be used to claim assets either locally on a system parachain, on the relay chain, or on any chain that supports the pallet-xcm `claimAssets` runtime call.
```

Comment:
I think using something like `palletXcm.claimAssets` is the best approach. I decided to remove the: `pallet-xcm` word from the paragraph because the API dynamically handles which pallet has the `claimAssets `call and uses it, such as `xcmPallet.claimAssets`, `polkadotXcm.claimAssets` and so on. 
I'll leave it like this:
```
Creates a local XCM transaction to retrieve trapped assets. This function can be used to claim assets either locally on a system parachain, on the relay chain, or on any chain that supports the `claimAssets` runtime call.
```